Apps/WebClient/src/ClientApp/src/constants/entryType.ts

```diff
@@ -97,6 +116,7 @@ entryTypeMap.set(EntryType.MedicationRequest, {
     icon: "file-medical",
     component: "MedicationRequestComponent",
     eventName: "special_authority",
+    enabled: (config) => isDatasetEnabled(config, "Medication"),
```

Comment:
This should be passing the dataset named "SpecialAuthorityRequest".  Once we've finished getting rid of the modules, we can update the enum values so these overrides won't be needed and we can just pass the EntryType as-is.

Reply:
We have a dataset called `specialAuthority`, is this the one?

Reply:
Yes, though I think it would be better if the dataset name included request.